test/TestLinearPremiumPriceOracle.js

```diff
@@ -0,0 +1,63 @@
+const ENS = artifacts.require('@ensdomains/ens/ENSRegistry');
+const BaseRegistrar = artifacts.require('./BaseRegistrarImplementation');
+const DummyOracle = artifacts.require('./DummyOracle');
+const LinearPremiumPriceOracle = artifacts.require('./LinearPremiumPriceOracle');
+
+const namehash = require('eth-ens-namehash');
+const sha3 = require('web3-utils').sha3;
+const toBN = require('web3-utils').toBN;
+
+contract('LinearPremiumPriceOracle', function (accounts) {
+    let priceOracle;
+
+    before(async () => {
+		ens = await ENS.new();
+		registrar = await BaseRegistrar.new(ens.address, namehash.hash('eth'));
+		await ens.setSubnodeOwner('0x0', sha3('eth'), registrar.address);
+		await registrar.addController(accounts[0]);
+
+        // Dummy oracle with 1 ETH == 2 USD
+        var dummyOracle = await DummyOracle.new(toBN(2000000000000000000));
+        // 4 attousd per second for 3 character names, 2 attousd per second for 4 character names,
+        // 1 attousd per second for longer names.
+        // Pricing premium starts out at 100 USD at expiry and decreases to 0 over 100k seconds (a bit over a day)
```

Comment:
Is this just a test figure or actually be used for real? If real, 100 USD premium is too low for high value name and 1 day curve is probably too steep.

Reply:
No, these are just convenient figures for testing.